requirements.txt

```diff
@@ -5,3 +5,4 @@ pbr
 termcolor==1.1.0
 setproctitle
 ordereddict==1.1
+ujson
```

Comment:
> Pip may not be available unfortunately

```
curl https://bootstrap.pypa.io/get-pip.py  > get-pip.py
python get-pip.py
```
what else ? ;)

> Another huge con, pip is not made to install configuration files and init script , it only install alignak as a lib, config file shipped are usually not in system path but those are "demo". 

well, this is relative problems.. as with having pip available : there are certainly solutions.
What I wanted to make more clear: I'm not against if you take consuming time to create deb/rpm/pkgs/.. but that will be less time usable for you to do other things.